### PR TITLE
Import Fix

### LINKED ISSUE #130: zero code change tf test fails
python  tests/zero_code_change/tensorflow_integration_tests.py 

fails with error 
Traceback (most recent call last):
  File "tests/zero_code_change/tensorflow_integration_tests.py", line 23, in <module>
    from tests.tensorflow.hooks.test_mirrored_strategy import test_basic
ModuleNotFoundError: No module named 'tests.tensorflow.hooks.test_mirrored_strategy'

### PR DESCRIPTION
**Pull request by jarednielsen**
### Description of changes:

Fixes #130 
isort moved around the imports, but I literally changed `from .tf_utils` to `from tf_utils`. Relative imports don't work when running a Python script. Tested by running `python tests/zero_code_change/tensorflow_integration_tests.py`

#### Style and formatting:

I have run `pre-commit install` to ensure that auto-formatting happens with every commit.

#### Issue number, if available

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Vikas-kum** [comment]
canyou also change config/tests.sh to run using python

**Comment by jarednielsen** [comment]
Done

**Comment by Vikas-kum** [comment]
Trigger 0 code change TF build and test - https://us-east-1.console.aws.amazon.com/codesuite/codebuild/projects/smdebug_tensorflow_zero_code_change_build/build/smdebug_tensorflow_zero_code_change_build%3Aeef9c8af-d608-4d2e-9c7c-96f351e717b8/log?region=us-east-1

**Comment by Vikas-kum** [comment]
@jarednielsen Failed build - https://us-east-1.console.aws.amazon.com/codesuite/codebuild/projects/smdebug_tensorflow_zero_code_change_build/build/smdebug_tensorflow_zero_code_change_build%3Aeef9c8af-d608-4d2e-9c7c-96f351e717b8/log?region=us-east-1

**Comment by jarednielsen** [comment]
Fixed the `import tests.tensorflow` issue: https://console.aws.amazon.com/codesuite/codebuild/projects/smdebug_tensorflow_zero_code_change_build/build/smdebug_tensorflow_zero_code_change_build%3A05b7f262-d166-4d2c-8ab9-8939ae65b150/log?region=us-east-1

Appears to have uncovered another bug (`AttributeError: module 'horovod.tensorflow' has no attribute 'size'`). Will investigate further tomorrow

**Comment by jarednielsen** [comment]
Success! https://console.aws.amazon.com/codesuite/codebuild/projects/smdebug_tensorflow_zero_code_change_build/build/smdebug_tensorflow_zero_code_change_build%3A45f143ed-7a97-405d-b4c4-c88cafa6f956/log?region=us-east-1

Don't have a folder structure `horovod/tensorflow/__init__.py` in your test folder, otherwise that will override the horovod package :)

File: config/buildspec_zero_code_change.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ phases:
     commands:
       - cd $CODEBUILD_SRC_DIR && chmod +x config/install_smdebug.sh && chmod +x config/check_smdebug_install.sh && ./config/install_smdebug.sh && cd ..
       - cd $RULES_CODEBUILD_SRC_DIR && python setup.py bdist_wheel --universal && pip install dist/*.whl && cd ..
-      - cd $CODEBUILD_SRC_DIR  && chmod +x config/tests.sh && ./config/tests.sh  && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
-      - cd $RULES_CODEBUILD_SRC_DIR && chmod +x config/tests.sh && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && ./config/tests.sh  && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
+      - cd $CODEBUILD_SRC_DIR  && chmod +x config/tests.sh && PYTHONPATH=. ./config/tests.sh  && mkdir -p upload/$CURRENT_COMMIT_PATH/wheels && cp ./dist/*.whl upload/$CURRENT_COMMIT_PATH/wheels && cd ..
```
Comment (Vikas-kum) [inline config/buildspec_zero_code_change.yml]: do we still need this ?

Comment (jarednielsen) [reply config/buildspec_zero_code_change.yml]: Yes, that addresses the first issue (importing tests.tensorflow)

**Comment by rahul003** [comment]
Why?

**Pull request merged**